eng/common/post-build/publish-using-darc.ps1

```diff
@@ -22,12 +22,12 @@ try {
   $optionalParams = [System.Collections.ArrayList]::new()
 
   if ("" -ne $ArtifactsPublishingAdditionalParameters) {
-    $optionalParams.Add("artifact-publishing-parameters") | Out-Null
+    $optionalParams.Add("--artifact-publishing-parameters") | Out-Null
```

Comment:
Won't this get overwritten with the next arcade update?

Reply:
It will, I've done a symmetric PR into arcade. This was to unblock til the update and to test it.